### DISCUSSION TITLE
r19base:/5.suspense

### DISCUSSION THREAD

**Discussion by giscus** [yangbo5207/react19feature#10] [General]
# tutorial/use/suspense

http://localhost:5173/tutorial/use/suspense

**Comment by yangbo5207** [comment]
欢迎各位道友添加我的微信好友 icanmeetu 交流学习心得

**Comment by dlgchg** [comment]
我来了

**Comment by wjgogogo** [comment]
> 一个很明显的差别就是 Suspense + use 的方式会自动帮助我们弃用第二次的请求数据。
这里应该是弃用第一次

**Comment by zzyx2021** [comment]
我怎么觉得以前的写法更好读呢

**Reply by yangbo5207** [reply]
这个得看个人理解

**Comment by panjiangyi** [comment]
`use` 在19里才有，但是18里的`<Suspense />`就已经能捕获抛出的promise错误显示fallback组件了。 那些状态管理库也早就支持`<Suspense />`的这种用法了。

**Reply by yangbo5207** [reply]
支持了，不过用法可能理解上不太一样
